src/overloads/arrays.jl

```diff
@@ -122,6 +122,19 @@ function LinearAlgebra.:\(
     return Ainv * B
 end
 
+function LinearAlgebra.:\(
+    A::AbstractMatrix, B::AbstractVecOrMat{T}
+) where {T<:AbstractTracer}
+    return inv(A) * B
+end
+
+function LinearAlgebra.:\(
+    A::AbstractMatrix{T1}, B::AbstractVecOrMat{T2}
+) where {T1<:AbstractTracer,T2<:AbstractTracer}
+    Ainv = LinearAlgebra.pinv(A)
+    return Ainv * B
+end
+
```

Comment:
You probably just want to call `second_order_or` and return a `Fill` here.

Reply:
I think this branch is something like `\(A::Abstract{<:AbstractTracer}, B::AbstractVecOrMat)` we defined before, the only difference is that here `B` is a matrix of gradient tracer right? Why don't we just directly use `pinv` here?

Reply:
Because we can skip the entire computation and assume that the index sets inside of the tracers all get mixed!

Reply:
I'm sorry, this is a bit of a non-trivial PR. I can probably find an hour tomorrow to push some changes.

Reply:
Oh, I see. I changed to this dispatch to use `second_order_or` and `Fill` here.